### PR TITLE
Implemented Standard Logger for Logging.

### PR DESCRIPTION
**Pull request by aditya-iitd**

### CODE REVIEW

File: standard_logger.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import logging
+
+
+def login():
```
Comment (KirillTim) [inline standard_logger.py]: What is the purpose of this method? How can it be used?

Comment (aditya-iitd) [inline standard_logger.py]: We can call this method whenever we want to log.
The output message i.e.. log info is a random message as of now

**Comment by marco-c** [comment]
Please create a new, clean, pull request, there are thirty unrelated commits here 😄

File: standard_logger.py (hunk 1 above)
Comment (marco-c) [inline standard_logger.py]: Can you explain in more detail what's the purpose here?
We could log with `logging.info` directly. Do you want to create a wrapper module for the logging module?

Comment (aditya-iitd) [inline standard_logger.py]: Yes i created a wrapper module because it then we would not have to write logger commands again and again.

Comment (marco-c) [inline standard_logger.py]: OK, a wrapper module sounds good. This method is strange though, as I can't see how you would use it from the rest of the code.
If we want a wrapper, there should be a way to perform logging at different severities (INFO, WARNING, ERROR, etc.).

Regarding the creation of a new clean pull request, if you don't know how to do it, feel free to ask me on IRC.

**Pull request closed without merging**